### ISSUE TITLE
clicable link  in bedfile in gffTags?

### ISSUE THREAD

**Issue by duartemolha** [igvteam/igv#1596]
Hi .. I was wondering if there is a way of adding a bedfile entry with actionable link to navigate to another feature /genomic coordinate in the opened IGV session

something like:

#gffTags
chr1    100  200 Name=Target1;link=http://localhost:60151/goto?locus=chr2:100-200

many thanks

**Comment by lbergelson** [comment]
That's an interesting idea.  I'm not sure I like having to edit your data to include a link though.  Maybe we could generally identify loci in the tooltip text and make them clickable.

**Comment by duartemolha** [comment]
Yes... That would be ideal... I just tried with the link because I know I
can get igv to navigate to a loci using these sort of links... So.i tried
embedding it on the tags :)

Having an actionable link detection on te tool tip that automatically
converts a chrN:start-end into a navigable link would be the best
